modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -87,6 +87,36 @@ object ElasticQuery {
   final def contains(field: String, value: String): WildcardQuery[Any] =
     Wildcard(field = field, value = s"*$value*", boost = None, caseInsensitive = None)
 
+  /**
+   * Constructs a type-safe instance of [[zio.elasticsearch.query.DisjunctionMax]] using the specified parameters.
+   *
+   * @param query
+   *   a mandatory query to be wrapped inside of disjunction max query
+   * @param queries
+   *   the rest of the queries to be wrapped inside of disjunction max query
+   * @tparam S
+   *   document for which field query is executed. An implicit `Schema` instance must be in scope
+   * @return
+   *   an instance of [[zio.elasticsearch.query.DisjunctionMax]] that represents the `disjunction max` query to be
+   *   performed.
+   */
+  final def disjunctionMax[S: Schema](query: ElasticQuery[S], queries: ElasticQuery[S]*): DisjunctionMaxQuery[S] =
+    DisjunctionMax[S](query = query, queries = Chunk.fromIterable(queries), tieBreaker = None)
```

Comment:
Can you merge query and queries here?
`queries = Chunk.fromIterable(queries).prepended(query)` or something like this?

Reply:
So then we will not have both `query` and `queries` in `DisjunctionMax`.